en/docs/using-wso2-identity-server/2-legged-OAuth-for-Securing-a-RESTful-Service.md

```diff
@@ -23,7 +23,7 @@ This page consists of the procedure to secure a RESTful service with
 4.  Go to WSO2 IS Management Console by pointing your browser to
     `                     https://localhost:9443/carbon/                   `
     .
-5.  [Register a user with WSO2 Identity Server](../../using-wso2-identity-server/configuring-users) by
+5.  [Register a user with WSO2 Identity Server](_Configuring_Users_) by
     providing a username and password.
```

Comment:
this link has been reverted back to the wrong link format

Reply:
Fixed